cypher-shell/src/test/java/org/neo4j/shell/state/BoltStateHandlerTest.java

```diff
@@ -0,0 +1,108 @@
+package org.neo4j.shell.state;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.neo4j.driver.v1.Transaction;
+import org.neo4j.shell.TestTransaction;
+import org.neo4j.shell.exception.CommandException;
+import org.neo4j.shell.log.Logger;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+public class BoltStateHandlerTest {
```

Comment:
If these tests can be changed with Mockito then we can remove a bunch of stubbed classes like TestTransaction, TestDriver, TestSession etc..